### ISSUE TITLE
My Catalog Table Headers Don't Announce Sorting Functionality

### ISSUE THREAD

**Issue by avaaec** [pressbooks/pressbooks#1780]
### Steps to Reproduce

1. Navigate to the My Catalog page
2. Navigate to the headings of the table in the middle of the page
3. Inspect the headings of the table and notice that there is no labeling that would indicate to a non-visual user that they are sortable. Additionally you can use a screen reader program to verify these findings. Currently NVDA announces this information for the "Title" column header:
```
Column 5, Title, Link
```
4. Notice that by default all sortable columns have aria-sort set to "none"

**Expected behavior:** 
The labeling for the column headers should be developed so that they are announced as buttons that sort the contents of the table. This can be accomplished by either giving each header a label that reads "sort by X" when aria-sort is set to none or by insuring that at least one of the headers has aria-sort set to truthy value at all times. Additionally all of the headers need to be developed so that they are not interpreted as links by assistive tech.
**Actual behavior:** 
The table headers include no indication that they can sort the contents of the table. Additionally by default none of them are given truthy values of aria-sort so non-visual users have no indication that the table is sortable. All they see is a series of links.

**Comment by avaaec** [comment]
This is Ava from the University of Washington - IT Accessible Technology Services team by the way. I will be filling a bunch more accessibility issues in the next few days so please let me know if there is anything I could be doing better to log these issues.

Take care and let me know if you have any questions!! :smile_cat:

**Comment by avaaec** [comment]
This is a UW ATS priority 2 issue

**Comment by SteelWagstaff** [comment]
Other tables in the interface have received improvements in https://github.com/pressbooks/pressbooks/commit/cc6fd00cb5a4833b01b18e4c5e8f88d53b51bd75 and https://github.com/pressbooks/pressbooks/issues/1633. These might serve as helpful models for changes needed here.

**Comment by SteelWagstaff** [comment]
The my catalog feature is one that we intend to retire.

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 6y 6m ago with no updates in the last 4d.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542

**Comment by greatislander** [comment]
After testing and finding that I could not reproduce this issue I discovered that the problem was resolved upstream by WordPress in the `WP_List_Table` class: https://core.trac.wordpress.org/ticket/32170. I consider this issue resolved.
